MSMetaEnhancer/app.py

```diff
@@ -59,31 +59,40 @@ async def annotate_spectra(self, converters, jobs=None, repeat=False):
         :param converters: given list of converters names
         :param jobs: list specifying jobs to be executed
         :param repeat: if some metadata was added, all jobs are executed again
+        :param monitor: given Monitor object to observe status of services
+        :param annotator: given Annotator object to run the actual annotation
         """
         async with aiohttp.ClientSession() as session:
             builder = ConverterBuilder()
             builder.validate_converters(converters)
             converters, web_converters = builder.build_converters(session, converters)
-            annotator = Annotator(converters)
 
-            # start converters status checker and wait for first status
-            monitor = Monitor(web_converters)
-            monitor.start()
-            monitor.first_check.wait()
-
-            # create all possible jobs if not given
-            if not jobs:
-                jobs = []
-                for converter in converters.values():
-                    jobs += converter.get_conversion_functions()
-            jobs = convert_to_jobs(jobs)
+            if annotator is None:
+                annotator = Annotator()
```

Comment:
I think the constructor call can actually be moved to the default value for the argument.